### ISSUE TITLE
Incorrect ids when multiple creds are created

### ISSUE THREAD

**Issue by ifellows** [rich-iannone/blastula#135]
Take a look at:

https://github.com/rich-iannone/blastula/blob/9998e41dca776eb4d35199f0f61ec48d8693f306/R/retrieve_credentials.R#L91

If your credentials look like:
```
cred_tbl <- structure(list(service = c("blastula-v1-gmail", "blastula-v1-outlook"
), username = c("xxx@xxx.com", "yyy@yyy.gov")), row.names = c(NA, 
-2L), class = c("tbl_df", "tbl", "data.frame"))
```

This function will return a table with incorrect ids
```
# A tibble: 2 x 3
  id    key_name            username          
  <chr> <chr>               <chr>             
1 gmail blastula-v1-gmail   xxx@xxx.com
2 gmail blastula-v1-outlook yyy@yyy.gov 
```

I think you want to replace this line with
```
creds_tbl <- tibble(
  id = unlist(lapply(creds_tbl$service, function(x) str_split(x, "-")[[1]][3])),
  key_name = creds_tbl$service,
  username = creds_tbl$username
  )
```

**Comment by Alternikaner** [comment]
Duplicate of #118

**Comment by ifellows** [comment]
Oh, thanks @Alternikaner . I didn't see that. Annoying to have duplicated running down the bug and finding a solution. @rich-iannone, is there an issue merging in @Alternikaner's solution? This appears to be a pretty critical bug.

**Comment by Alternikaner** [comment]
No apologies necessary, @rich-iannone. Thanks for your hard work.
(And good luck with the Outlook encoding/rendering weirdness - it's bested me thus far.)

**Comment by Alternikaner** [comment]
Fixed by #119
